### ISSUE TITLE
Winlogbeat not parsing config files with leading BOM

### ISSUE THREAD

**Issue by StianOvrevage** [elastic/beats#2354]
Forked from issue #1349.

Winlogbeat will not start if the config file contains a leading BOM. (Which Windiws' notepad.exe prepends)

C:\communitor\winlogbeat>winlogbeat.exe -configtest
Loading config file error: YAML config parsing failed on C:\winlogbeat\winlogbeat.yml: yaml: invalid leading UTF-8 octet. Exiting.

This, ironically, makes you unable to edit the Windows log beat ON Windows.

**Comment by andrewkroh** [comment]
Thanks for reporting. What version are you using? I assume this problem affects other Beats too. Have you tried any of the others to see if they are affected?

**Comment by StianOvrevage** [comment]
I just tested with version 1.2.3 and the error is there. 

A small correction: if you edit the original config file that ships in the zip file everything is fine. But if you create a new file in notepad it might save the file as UTF8 BOM. Sometimes it saves in ANSI. Guessing it depends on the characters used in the file.

I have also tested topbeat, packetbeat and filebeat and they all seem to work with UTF8 BOM and ANSI encoding without problems.
